packages/block-library/src/heading/test/index.native.js

```diff
@@ -135,10 +136,23 @@ describe( 'Heading block', () => {
 			)
 		).toBeVisible();
 	} );
+	it( 'changes heading level', async () => {
```

Comment:
Tiniest nit ever, but just missing a new line here!

```suggestion

	it( 'changes heading level', async () => {
```